app/src/main/kotlin/com/wire/android/mapper/MessageContentMapper.kt

```diff
@@ -157,7 +157,13 @@ class MessageContentMapper @Inject constructor(
             val assetMessageContentMetadata = AssetMessageContentMetadata(content.value)
             toUIMessageContent(assetMessageContentMetadata, message, sender)
         }
-
+        is MessageContent.Knock -> UIMessageContent.SystemMessage.Knock( // TODO should we move Knock to System message also in DAO layer?
```

Comment:
On collective we discussed that and decided that ping should be considered as a regular message but should look like a system message, I think we can remove this TODO 😉

Reply:
I will refactor It after meeting with PM and Designer